tests/phpstan/phpstan-latest.neon

```diff
@@ -2,8 +2,9 @@ includes:
 	- %currentWorkingDirectory%/tests/phpstan/phpstan.neon
 
 parameters:
+	excludes_analyse:
+		- ./../../classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php
```

Comment:
Why is it ignored here? I think it make sense to test it against the PrestaShop version it is supposed to work with?

Reply:
That's maybe covering another issue then? It should be able to see the core dependencies IMO 🤔

Reply:
I might be missing something about how phpstan works, how would you do that, with comments ?